Clockwise/InMemoryCircuitBreakerBroker.cs

```diff
@@ -11,11 +11,14 @@ public sealed class InMemoryCircuitBreakerBroker : ICircuitBreakerBroker
         private readonly ConcurrentDictionary<Type, CircuitBreakerStoragePartition> partitions = new ConcurrentDictionary<Type, CircuitBreakerStoragePartition>();
         private class CircuitBreakerStoragePartition
         {
+            private static readonly Logger logger = Logger<CircuitBreakerStoragePartition>.Log;
             private readonly ConcurrentSet<CircuitBreakerBrokerSubscriber> subscribers;
             private CircuitBreakerStateDescriptor stateDescriptor;
+            private readonly string key;
 
-            public CircuitBreakerStoragePartition()
+            public CircuitBreakerStoragePartition(Type type)
```

Comment:
This should be a string since it's only used for its name. It should also have a guard clause.